composer/workflows/connections.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
```

Comment:
No need to update the copyright. It should reflect the year that the file was created.

Reply:
Reverted 👍